docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1388,7 +1388,10 @@ tag::state-datafeed[]
 The status of the {dfeed}, which can be one of the following values:
 +
 --
+* `starting`: The {dfeed} has been requested to start but has not yet started.
 * `started`: The {dfeed} is actively receiving data.
+* `stopping`: The {dfeed} has been requested to stop gracefully and is
+completing its final action.
```

Comment:
The `stopping` state is not introduced by this PR - it has existed for a couple of years.  But I thought I'd add it while I was modifying the list.

(This PR exposes the `starting` state externally for the first time, so it's understandable it wasn't previously documented.)